Source/RapyutaSimulationPlugins/Private/RRROS2GameMode.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020-2021 Rapyuta Robotics Co., Ltd.
```

Comment:
Can you name it as ROS2GameMode?

Reply:
Ah sure, I guess since this is a public repo, so we should not prefix class name right?